tests/phpunit/tests/option/option.php

```diff
@@ -478,4 +478,23 @@ public function test_update_option_with_autoload_change_yes_to_no() {
 		delete_option( 'foo' );
 		$this->assertFalse( get_option( 'foo' ) );
 	}
+
+	/**
+	 * Tests that calling delete_option() update notoptions when option deleted.
+	 *
+	 * @ticket 61484
+	 *
+	 * @covers ::delete_option
+	 */
+	public function test_check_delete_option_updates_notoptions() {
+		add_option( 'foo', 'value1' );
+
+		delete_option( 'foo' );
+
+		$before = get_num_queries();
+		get_option( 'foo' );
+		$after = get_num_queries();
+
+		$this->assertSame( $after, $before, 'The notoptions cache was not hit.' );
```

Comment:
This is to make it clearer that no queries are expected. If it fails it will produce the error that 1 does not match 0 rather than something along the lines of 1234 does not match 1233.

```suggestion
		$queries = get_num_queries() - $before;

		$this->assertSame( 0, $queries, 'get_option should not make any database queries.' );
```